package.json

```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "bignumber.js": "^6.0.0",
     "carbon-components": "8.16.8",
-    "carbon-components-react": "5.31.1",
+    "carbon-components-react": "5.32.0",
```

Comment:
Is there any significant updates in this version? If yes, then please create issue to update this package in other our repos. We should have the same version everywhere.

Reply:
I Might have upgraded to get one of the components to render properly, but I did this early so I can't remember exactly which one. I will downgrade, see if it works and if so I will keep it at 5.31.1. But if I need 5.32.0 , I will open an issue

Reply:
yes 5.31.1 works for me , I will fix this

Reply:
fixed